src/main/java/org/example/TaskManager.java

```diff
@@ -0,0 +1,133 @@
+package org.example;
+import org.apache.commons.lang3.*;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class TaskManager {
+    static File file = new File("tasks.csv");
+    static String[][] tasks;
+    static final String[] OPTIONS = {"add", "remove", "list", "exit"};
```

Comment:
CLEAN-CODE: globalni promenne prosim dat pristupne pouze z tridy TaskManager, pridat private pred promenne, plus novy radek mezi definici globalnich promennych a metod.
```suggestion
    final static File file = new File("tasks.csv");
    final static String[][] tasks;
    final static final String[] OPTIONS = {"add", "remove", "list", "exit"};
    
```

Reply:
opraveno